gsy_framework/utils.py

```diff
@@ -596,3 +597,8 @@ def use_default_if_null(input_value: Union[int, str, float], default_value: Unio
                         ) -> Union[int, str, float]:
     """Return input_value if not null, else return provided default_value"""
     return input_value if input_value not in NULL_VALUES else default_value
+
+
+def is_canary_network():
+    """Return if the GlobalConfig is set up for a Canary Network"""
+    return GlobalConfig.CONFIG_TYPE == ConfigurationType.CANARY_NETWORK.value
```

Comment:
How about moving this method in the `GlobalConfig` class ? That way it will be grouped together with the configuration and will not be part of the generic `utils` module

Reply:
Good idea, let me add this.